src/ceph-detect-init/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-set(CEPH_DETECT_INIT_VIRTUALENV ${CMAKE_BINARY_DIR}/ceph-detect-init-virtualenv)
+set(CEPH_DETECT_INIT_VIRTUALENV /tmp/ceph-detect-init-virtualenv)
```

Comment:
can we read "/tmp" from `ENV{TMPDIR}` and default to "/tmp" if it's empty or not set?